packages/block/src/block.ts

```diff
@@ -14,7 +14,7 @@ export class Block {
   public readonly uncleHeaders: BlockHeader[] = []
   public readonly txTrie = new Trie()
 
-  private readonly _common: Common
+  public readonly _common: Common
```

Comment:
A question for context: is the idea to change `_common` visibility so it can be available on the tests?

Reply:
yeah in general I agree with this so we don't have to keep doing `(block as any)._common` anymore 😄

Reply:
Yeah, that is for the tests. Always not sure if this is the best way to tackle, but then do out of lack on alternative ideas on practical paths here.